.concourse/tasks/linux-test.yml

```diff
@@ -16,8 +16,9 @@ run:
   - |
     rustc --version
     export RUSTFLAGS="-D warnings"
-    cargo check --all --features sdl_controller,profiler
+    cargo check --all --features sdl_controller,profiler,gltf
```

Comment:
`--all-features` here as well?

Reply:
would not work on nightly. Torkleyy made a change to support it in another PR.